src/bases/EnclaveIdentityDao.sol

```diff
@@ -125,20 +126,34 @@ abstract contract EnclaveIdentityDao is DaoBase, SigVerifyBase {
         uint256 id,
         uint256 version,
         EnclaveIdentityJsonObj calldata enclaveIdentityObj
-    ) private view returns (bytes memory reqData) {
+    ) private view returns (bytes32 key, bytes memory reqData) {
         IdentityObj memory identity = EnclaveIdentityLib.parseIdentityString(enclaveIdentityObj.identityStr);
         if (id != uint256(identity.id)) {
             revert Enclave_Id_Mismatch();
         }
 
-        if (id == uint256(EnclaveId.TD_QE) && version != 4) {
+        if (id == uint256(EnclaveId.TD_QE) && version < 4) {
```

Comment:
I'll constrain it to 4 and 5, cause at this current moment all other versions are invalid and we have no idea in the future Intel will actually support TD_QE in version 6.

Reply:
[eb1a642](https://github.com/automata-network/automata-on-chain-pccs/pull/15/commits/eb1a64271d8a4d9ed9cbb4ed69f6054ff20ff8a5)